app/cdap/components/hydrator/components/TopPanel/ActionButtons/styles.ts

```diff
@@ -106,3 +107,7 @@ export const RunTimeSpan = styled.span`
   font-weight: 500;
   vertical-align: middle;
 `;
+
+export const HiddenInput = styled.input`
+  display: none !important;
+`; // beating specificity. The file input needs to be hidden at all times
```

Comment:
@njbriggs are we okay with using `!important`? 

another way of doing this is 
```
&&& {
    display: none;
}
```

which seems to be the recommended way of doing this in styled components https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

Reply:
great point! Updated

Reply:
Sounds like we should start using this